index.js

```diff
@@ -49,12 +49,24 @@ module.exports = app => {
 			return
 		}
 
-		// 2. command syntax checks
+		// 2.1 command action checks
+		const action = commands.arguments.startsWith('amend')
+			? 'amend'
+			: commands.arguments.startsWith('fixup')
+				? 'fixup'
+				: false
```

Comment:
Ternary-ception

Reply:
So? :p

Reply:
You should add another to make it a ternary ternary.